### ISSUE TITLE
Polly's describe_voices should support pagination

### ISSUE THREAD

**Issue by frasern** [boto/botocore#1261]
According to the API documentation, the [DescribeVoices](http://docs.aws.amazon.com/polly/latest/dg/API_DescribeVoices.html) command for Polly supports pagination.

However, botocore doesn't seem to support it:

```python
>>> import botocore.session
>>> botocore.__version__
'1.5.95'
>>> session = botocore.session.get_session()
>>> client = session.create_client('polly')
>>> client.can_paginate('describe_voices')
False
>>> client.get_paginator('describe_voices')
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    client.get_paginator('describe_voices')
  File "/home/fraser/venv/lib/python2.7/site-packages/botocore/client.py", line 654, in get_paginator
    raise OperationNotPageableError(operation_name=operation_name)
OperationNotPageableError: Operation cannot be paginated: describe_voices
```

At present the number of Polly voices is small enough that pagination isn't needed and all of them can be returned in a single request. However, presuming more voices will be added in the future, it would be good for botocore to support pagination here.

**Comment by joguSD** [comment]
Thanks for the report. I'll mark this as a feature request.

**Comment by swetashre** [comment]
I am closing this issue as this has been fixed with this PR [#1308](https://github.com/boto/botocore/pull/1308).
